db/models.py

```diff
@@ -1 +1,31 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, related_name="skill_set", on_delete=models.CASCADE
```

Comment:
The `related_name` for the `race` ForeignKey in the `Skill` model should be pluralized to `skills` to correctly represent the reverse relationship.